Audible-Altimeter/src/audio/src/audio_player.cpp

```diff
@@ -0,0 +1,25 @@
+#include "audio_player.h"
+#include "sample_id.h"
+#include "samples/audio_samples.h"
+#include <array>
+#include <cstdint>
+#include <cstddef>
+
+Audio::Audio(AudioDriver const& impl) : m_impl{impl}{}
+
+bool Audio::play(audio_sample_id_t sample_id) const {
+    assert(sample_id < AUDIO_SAMPLE_ID::NUM_SAMPLES);
+    bool rval { false };
+    if (sample_id < AUDIO_SAMPLE_ID::NUM_SAMPLES) {
+        rval = m_impl.play(sample_lookup[sample_id].location, sample_lookup[sample_id].size);
+    }
```

Comment:
```suggestion
    } else {
        assert(false);
    }
```
Investigate how to ensure that asserts become no-ops during release builds